addons/sale/views/sale_order_views.xml

```diff
@@ -226,11 +226,21 @@
             </div>
             <sheet>
                 <div class="oe_button_box" name="button_box">
+                    <button name="action_view_invoice"
+                        type="object"
+                        class="oe_stat_button text-danger"
+                        icon="fa-pencil-square-o"
+                        attrs="{'invisible': ['|', ('invoice_count', '=', 0), ('any_draft_invoice', '=', False)]}">
+                        <div class="o_field_widget o_stat_info">
+                            <field name="invoice_count" nolabel="1"/>
+                            <span class="o_stat_text">Invoices</span>
+                        </div>
+                    </button>
```

Comment:
Is this really necessary only to show some text in red ?

Reply:
As draft down payments are taken into account in the calculation of the amount when generating final invoice, the idea here is to draw the user attention to the fact that some invoice still are draft.

Reply:
I'd prefer some kind of warning in the invoice wizard instead of a duplicated statbutton (and yet another field on sales orders).
The problem with 'nearly duplicated' view elements is that we sometimes get user tickets saying 'oh, i customized this with studio, but in this situation, it doesn't work as expected' because the element they modified or hooked their customization on is not always the one shown.

Reply:
> I'd prefer some kind of warning in the invoice wizard instead of a duplicated statbutton (and yet another field on sales orders). The problem with 'nearly duplicated' view elements is that we sometimes get user tickets saying 'oh, i customized this with studio, but in this situation, it doesn't work as expected' because the element they modified or hooked their customization on is not always the one shown.

Ok, make sense.

Reply:
@tsb-odoo